test/gtest/uct/test_p2p_am.cc

```diff
@@ -551,7 +551,8 @@ const unsigned uct_p2p_am_misc::RX_QUEUE_LEN = 64;
 
 UCS_TEST_SKIP_COND_P(uct_p2p_am_misc, no_rx_buffs, RUNNING_ON_VALGRIND) {
 
-    mapped_buffer sendbuf(10 * sizeof(uint64_t), SEED1, sender());
+    mapped_buffer sendbuf(ucs_min(sender().iface_attr().cap.am.max_short,
```

Comment:
why need this change?

Reply:
why is to fit into short required here?

Reply:
> why is to fit into short required here?

It will fail w/ the following error:
`UCX  ERROR Invalid am_short length: 80 (expected: <= 64)`

Reply:
ic, it uses uct am_short for sending